merlion/models/utils/rolling_window_dataset.py

```diff
@@ -0,0 +1,192 @@
+import numpy as np
+from typing import Optional
+from merlion.utils.time_series import TimeSeries
+
+
+class RollingWindowDataset:
+
+    def __init__(
+            self,
+            data: TimeSeries,
```

Comment:
Can you allow `data` to be either `TimeSeries` or `pd.DataFrame`?

Reply:
absolutely

Reply:
Done